### DISCUSSION TITLE
Icons do not animate under IOS 15.5

### DISCUSSION THREAD

**Discussion by alexwala** [FortAwesome/Font-Awesome#19148] [Q&A] [answered]
Opening the following page on an iPhone (IOS 15.5)

[Animating Icons](https://fontawesome.com/v6/docs/web/style/animate)

currently they do not animate on the iPhone, in contrast to e.g. chrome on windows

see also oznu/homebridge-config-ui-x#1331

has anyone observed this?

thx.

**Comment by tagliala** [comment, marked as answer]
Hi!

Thanks for being part of the Font Awesome Community.

Probably the option "Prefer reduced motion" is set at OS level, so Font Awesome is acting in the described way

https://support.apple.com/guide/iphone/motion-iph0b691d3ed/ios

**Reply by alexwala** [reply]
thanks, yes, that was the right hint !
interessting: I seem to remember that it still worked with ios 15.0 even though reduce motion was activated
